### PR TITLE
Navigation template

### LINKED ISSUE #13836: template navigation is not longer working
### Description

Template navigation is not longer working in rn 0.44

### Reproduction Steps

`react-native init template --navigation`

### PR DESCRIPTION
**Pull request by agrcrobles**
## Motivation

Fixes #13836

### CODE REVIEW

File: local-cli/templates/HelloNavigation/views/chat/ChatListScreen.js [hunk 1]
```diff
@@ -16,18 +16,14 @@ export default class ChatListScreen extends Component {
...
-    tabBar: {
-      icon: ({ tintColor }) => (
-        <Image
-          // Using react-native-vector-icons works here too
-          source={require('./chat-icon.png')}
-          style={[styles.icon, {tintColor: tintColor}]}
-        />
-      ),
-    },
+    header: Platform.OS === 'ios' ? undefined : null,
```
Comment (mkonicek) [inline local-cli/templates/HelloNavigation/views/chat/ChatListScreen.js]: Is this really the API now? Undefined vs null?

**Comment by mkonicek** [comment]
Thanks for the fix! I don't work on RN anymore unfortunately - @skevy does this look good to you?

File: local-cli/templates/HelloNavigation/dependencies.json [hunk 2]
```diff
@@ -1,3 +1,3 @@
 {
-  "react-navigation": "1.0.0-beta.5"
+  "react-navigation": "1.0.0-beta.9"
```
Comment (skevy) [inline local-cli/templates/HelloNavigation/dependencies.json]: Could you update this to beta.11, and ensure that everything still works as you expect? :)

Comment (agrcrobles) [inline local-cli/templates/HelloNavigation/dependencies.json]: sure, I am on vacation right now i will take a look once back

**Pull request closed without merging**